src/routes/+layout.svelte

```diff
@@ -13,7 +13,7 @@
 </script>
 
 <QueryClientProvider client={data.queryClient}>
-  <div class="flex min-h-screen flex-col">
+  <div class="flex min-h-screen flex-col overflow-hidden">
```

Comment:
I'm not sure if we should add `overflow-hidden` to the layout and therefor the entire app, or if we should scope the hidden overflow to the one route that is affected only. What do you think?

Reply:
I don't think specifying overflow-hidden in layout will cause any problem in UI. Moreover we can add `overflow-x-hidden` instead of `overflow-hidden`, so it only hides the scroll bar in x-axis.